infogami/infobase/client.py

```diff
@@ -159,7 +159,12 @@ def request(self, sitename, path, method='GET', data=None):
         headers['X-REMOTE-IP'] = web.ctx.get('ip') or ''
 
         try:
-            response = requests.request(method, path, data=data, headers=headers)
+            response = requests.request(
+                method,
+                f'http://{self.base_url}{path}',
+                data=data,
+                headers=headers
```

Comment:
Try to appease black

```suggestion
                method, f'http://{self.base_url}{path}', data=data, headers=headers
```